### ISSUE TITLE
Not Loading in Cura 5.4.0

### ISSUE THREAD

**Issue by ledgend78** [smartavionics/RawMouse#24]
When I try to load up Cura with this plugin, I get the error "The Plugin RawMouse Could not be loaded"

**Comment by DivingDuck** [comment]
It works works for Cura 5.4. Simply install the latest version from this repository here: https://github.com/smartavionics/RawMouse/releases

**Comment by Evac72** [comment]
![Screenshot 2023-10-22 184256](https://github.com/smartavionics/RawMouse/assets/125583253/e0439a4d-f944-470b-82b9-8cbf3adc0474)
i am also having issue installing it for cura 5.4.0 and according to the log seems to be an sdk version issue

**Comment by smartavionics** [comment]
And you are using the latest version of RawMouse?

**Comment by Evac72** [comment]
> And you are using the latest version of RawMouse?

Yes, I believe, 1.2.1 right? That is the one labeled as the latest on here. Cura is 5.4.0 I did a fresh install of Cura too.

**Comment by smartavionics** [comment]
Hmm, the last couple of  RawMouse releases do support SDK level 8 so I wonder if somehow you are loading an old version. Where did you install RawMouse?

**Comment by ledgend78** [comment]
I got it working

**Comment by Evac72** [comment]
> Hmm, the last couple of RawMouse releases do support SDK level 8 so I wonder if somehow you are loading an old version. Where did you install RawMouse?

![image](https://github.com/smartavionics/RawMouse/assets/125583253/9ab35378-68b6-418b-b209-f9beb81f852d)
installed it in the plugin section

**Comment by smartavionics** [comment]
What other plugins are in the plugins folder? Is there another, earlier, RawMouse?

**Comment by Evac72** [comment]
![image](https://github.com/smartavionics/RawMouse/assets/125583253/768c356b-d736-4026-a49d-25eed207c6d0)


> What other plugins are in the plugins folder? Is there another, earlier, RawMouse?

![image](https://github.com/smartavionics/RawMouse/assets/125583253/df744ffd-bbdd-47f1-82f1-1641d8ccc468)
that all my plug-ins in the folder. Also there is no other cura version installed just this one

**Comment by Evac72** [comment]
> What other plugins are in the plugins folder? Is there another, earlier, RawMouse?

Good News it working now, not sure why tho, i just given my pc a restart and it works with nothing changed

**Comment by smartavionics** [comment]
Ah, good!

**Comment by Evac72** [comment]
> Ah, good!

Thank you for the help and creating this plugin. Strange it needed a full pc restart but if it works it works I am not going to question why it works lol.
